packaging/elasticsearch_plugin.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import os
+
+from ansible.module_utils.basic import *
```

Comment:
so this is not a real import, we recommend you add it to the bottom of the module to allow for easier debugging.